configuration/eslint/eslint.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   parser: 'babel-eslint',
-  extends: 'airbnb',
+  extends: 'formidable/configurations/es6-react',
   env: {
     browser: true,
```

Comment:
Future work -- this eslint should be split and more nuanced. It combines both the Test and App eslint configs. In the future, remove the test stuff and add `test/.eslintrc` or something...